crates/bevy_pbr/src/deferred/deferred_lighting.wgsl

```diff
@@ -87,7 +88,7 @@ fn fragment(in: FullscreenVertexOutput) -> @location(0) vec4<f32> {
 #endif
 #endif
 #ifdef PREMULTIPLY_ALPHA
-    output_color = pbr_functions::premultiply_alpha(material.flags, output_color);
+    output_color = pbr_functions::premultiply_alpha(pbr_input.material.flags, output_color);
```

Comment:
This never results in an error because the code is unreachable. We should actually remove pbr_functions::premultiply_alpha altogether as deferred is only used with opaque and alpha mask materials, while premultiply_alpha is only for alpha blend.

Reply:
removed